### DISCUSSION TITLE
Found the motif exact position in the peak

### DISCUSSION THREAD

**Discussion by hamidsta** [stuart-lab/signac#1156] [Q&A] [answered]
Hello, i wonder if there is a way to know the exact motif position after running findmotif() . Actually the ouput of the function is a dataframe with the entire peaks enrich for the motif . 
Is there any ways to know were the motif is located on the peaks ? 

Thanks

**Comment by timoast** [comment, marked as answer]
After running the `AddMotifs()` function, there will be a`GRangesList` object of motif positions stored in the motif object. You can access it using the `GetMotifData()` function with `slot = "positions"`
